app/views/RoomActionsView/index.js

```diff
@@ -320,6 +330,31 @@ class RoomActionsView extends React.Component {
 			setLoadingInvite(true);
 			await RocketChat.addUsersToRoom(rid);
 			navigation.pop();
+		} catch (e) {
+			log(e);
+			Alert.alert(
+				I18n.t('Confirmation'),
+				I18n.t('Removing_user_from_this_Team'),
+				[
+					{
+						text: I18n.t('OK'),
+						style: 'cancel'
+					}
+				],
+				{ cancelable: false }
+			);
```

Comment:
The function is in `addUser` and the catch raises a confirmation alert saying `removing user from this team`.
Can you explain?

Reply:
No reason, removed